internal/blst/non_cgo.go

```diff
@@ -0,0 +1,4 @@
+//go:build !cgo && no_cgo
+// +build !cgo,no_cgo
```

Comment:
What is the reasoning behind the additional `no_cgo` build tag? Isn't `!cgo` equivalent?

Reply:
Good question. I first only used `!cgo`, it is enough as you said. However, in the Discord thread, we worried this can be an issue for some projects building/releasing mistakenly without cgo, the build silently succeeds and the app crashes/panics later if any non-supported function is called. 
Specifying `no-cgo` when building avoids these mistakes and confirms the intention of the build. 
It is explained in the [readme](https://github.com/onflow/crypto/tree/tarak/build-without-cgo?tab=readme-ov-file#build-without-cgo) too.

Reply:
Ah, I see! Thanks for the great explanation!